subcommand/mesh-init/command.go

```diff
@@ -1,89 +1,61 @@
 package meshinit
```

Comment:
Extracting validations and constructing Consul types makes this much more readable 😄